src/benchmarklib/tpcc/procedures/tpcc_delivery.hpp

```diff
@@ -0,0 +1,22 @@
+#include "abstract_tpcc_procedure.hpp"
+
+namespace opossum {
+
```

Comment:
Not sure what that would accomplish. Unless we duplicate the specification for each procedure, this would be incomplete and readers would be lure into a false sense of security having "understood" the procedure. If you feel the need of diving into the procedures, having a look at the specification should be part of that.